frontend/src/component/Account/UserSpacesConfig.jsx

```diff
@@ -42,22 +56,18 @@ export const UserSpacesConfig = (props) => {
       { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.MODIFIED, handler: handleMemberModified },
       { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.DELETED, handler: handleMemberDeleted }
     ])
-
-    return () => {
-      // unregisterLiveMessageHandlerList
-    }
-  }, [])
+  }, [spaceList])
```

Comment:
You can merge theses two `uesEffect` (line 53-59 with 61-84) since they are both triggered on `spaceList`.
But since one is dedicated to `registerLiveMessageHandler` I'm fine keeping two separates `useEffect`

Reply:
I prefer to keep this way, I hesitate during dev, but I think it's more clear like that. It's ok for you ?

Reply:
Sure thing!